src/nl/surf/eduhub_rio_mapper/rio/aangeboden_opleiding.clj

```diff
@@ -117,7 +117,7 @@
   "Given a course or program, a rio-consumer object and an id, return a function.
    This function, given a attribute name from the RIO namespace, returns the corresponding value from the course or program,
    translated if necessary to the RIO domain."
-  [{:keys [offerings level modeOfStudy sector timelineOverrides fieldsOfStudy] :as course-program}
+  [{:keys [offerings level modeOfStudy sector periods fieldsOfStudy] :as course-program}
```

Comment:
Er is geen veld "periods" in een course of program. Haal die code uit `program->aangeboden-opleiding` hier heen dan kan course-program hier gewoon een echt course of program zijn.

Je kan bijvoorbeeld hier op regel 124

```clojure
(let [duration-map ...
      periods ...] ...)
```

doen, dan hoef je niks in die course-program aan te passen